### PR TITLE
changed the font style to  modern one

### PR DESCRIPTION
**Pull request by mcbobby7**
i updated the font to a more modern on that can be read easily

### CODE REVIEW

**Review by EngrEric — COMMENTED** [review]
the font size is small and make more research on the better font to use.

File: src/components/app/components/styles.css [hunk 1]
```diff
@@ -2,3 +2,11 @@
...
     text-align: center;
 }
+
+body {
+    font-family: 'Work Sans', sans-serif;
+   }
+   
+   h1, h2, h3, h4, h5, h6 {
+    font-family: 'Crimson Text', serif;
+   }
```
Comment (Way2nnadi) [inline src/components/app/components/styles.css:12]: The formatting here is wrong. Keep the formatting consistent
```
body {
    font-family: 'Work Sans', sans-serif;
}

h1, h2, h3, h4, h5, h6 {
    font-family: 'Crimson Text', serif;
} 
```

**Comment by enyeInc** [comment]
@mcbobby7 is this PR still needed?

**Review by enyeInc — COMMENTED** [review]
close this PR

**Comment by EngrEric** [comment]
@mcbobby7  find better modern fonts and replace this one. demo your work @ the slack channel for general approval

**Pull request closed without merging**